cmd/ui/src/views/Explore/ContextMenu/ContextMenu.test.tsx

```diff
@@ -36,33 +37,40 @@ describe('ContextMenu', async () => {
     );
 
     beforeAll(() => server.listen());
-    beforeEach(async () => {
-        await act(async () => {
-            render(<ContextMenu contextMenu={{ mouseX: 0, mouseY: 0 }} handleClose={vi.fn()} />, {
-                initialState: {
-                    entityinfo: {
-                        selectedNode: {
-                            name: 'foo',
-                            id: '1234',
-                            type: 'User',
-                        },
-                    },
-                    assetgroups: {
-                        assetGroups: [
-                            { tag: 'owned', id: 1 },
-                            { tag: 'admin_tier_0', id: 2 },
-                        ],
-                    },
+    afterEach(() => server.resetHandlers());
+    afterAll(() => server.close());
+
+    const setup = async (permissions?: Permission[]) => {
+        let initialState: object = {
```

Comment:
It would be cool if we could use something like `Partial<AppState>` for the type here. That would require a lot of out of scope changes though so am mostly just commenting because the `object` type caught my eye.

Reply:
wow you did it and didn't have to make a ton of changes! bravo 👏